src/app/zwiazek/historia-zwiazku/_components/UnionSubsection.tsx

```diff
@@ -0,0 +1,31 @@
+import { BulletpointList } from "./BulletpointList";
+
+interface Subsection {
+  subtitle: string;
+  paragraph: string[];
+  bulletpoints?: string[];
+}
+
+export const UnionSubsection = ({
+  subtitle,
+  paragraph,
+  bulletpoints,
+}: Subsection) => (
+  <section className="pt-5">
+    <h3 className="pb-4 font-lora text-24 font-bold leading-9 text-greenMain contrast:text-yellowContrast">
+      {subtitle}
+    </h3>
+    {paragraph.map((text, index) => (
+      <p key={index} className="pb-5">
+        {text}
+      </p>
+    ))}
+    {bulletpoints && (
+      <ul>
```

Comment:
Could be just a list that get's the `bulletpoints` and renders nothing if there are no bulletpoints.

Or just conditional render. `{bulletpoints ? <BulletpointList bulletpoints={bulletpoints} /> : null}`

```tsx
interface Props {
  bulletpoints: string[];
}

const BulletpointList = ({ bulletpoints }: Props) => (
  <ul>
    {bulletpoints.map((text, i) => (
      <li className="flex flex-row gap-6 pb-5">
        <IconBullet
          className="flex-shrink-0 text-redMain contrast:text-yellowContrast"
          aria-hidden="true"
        />
        <p>{text}</p>
      </li>
    ))}
  </ul>
);
```